frontend/src/components/PostRecipe.jsx

```diff
@@ -1,308 +1,598 @@
-                    <div className="recipe-form">
-                        <h2 id='title'>Post recipe</h2>
-    
-                        <div className="input-section">
-                            <label id='subtitle'>Recipe Name</label>
-                            <input id='input_postRecipe' 
-                                type="text" 
-                                value={recipeName} 
-                                onChange={(e) => setRecipeName(e.target.value)} 
-                                placeholder="Enter Recipe Name" 
-                            />
-                        </div>
-                        
-                        <div className="input-subsection">
-                            <div className="ingredients-section">
-                                <label>INGREDIENTS</label>
-                                <div className="ingredient-list">
-                                    {ingredients.map((ingredient, index) => (
-                                        <div key={index} className="ingredient-item">
-                                            <span>{ingredient.name} - {ingredient.quantity} {ingredient.unit}</span>
-                                            <button id='buttons_postRecipe'  onClick={() => handleIngredientDelete(index)}>X</button>
-                                        </div>
-                                    ))}
-                                </div>
-                                <div className="ingredient-input-group">
-                                    <input id='input_postRecipe-ingredient' 
-                                        type="text" 
-                                        ref={ingredientNameRef}
-                                        placeholder="Ingredient Name"
-                                    />
-    
-                                    <input id='input_postRecipe' 
-                                        type="number" 
-                                        ref={ingredientQuantityRef}
-                                        placeholder="Quantity"
-                                    />
-                                    <select className="ingredient-unit-select"
-                                        ref={ingredientUnitRef}
+              >
+                <Container
+                  id="recipe-container"
+                  className="mt-3 rounded box-shadow"
+                >
+                  <Row>
+                    <Col xs={12}>
+                      <div className="ingredient-list">
+                        <TransitionGroup component={null}>
+                          {ingredients.map((ingredient, index) => (
+                            <CSSTransition
+                              key={index}
+                              timeout={500}
+                              classNames="ingredient-fade"
+                            >
+                              <div key={index}>
```

Comment:
Redundant div, you can move the key property to the Row below, same reason as the previous comment

Reply:
Thanks. Later I'll make the suggested changes.